net-im/discord-bin/discord-bin-0.0.20-r2.ebuild

```diff
@@ -21,41 +21,47 @@ SRC_URI="https://dl.discordapp.net/apps/linux/${MY_PV}/${MY_PN}-${MY_PV}.tar.gz"
 LICENSE="all-rights-reserved"
 SLOT="0"
 KEYWORDS="amd64"
+RESTRICT="bindist mirror strip test"
+IUSE="+seccomp swiftshader system-ffmpeg"
 
-# libXScrnSaver is used through dlopen (bug #825370)
 RDEPEND="
-	|| (
-		>=app-accessibility/at-spi2-core-2.46.0:2
-		( app-accessibility/at-spi2-atk dev-libs/atk )
-	)
+		|| (
+			>=app-accessibility/at-spi2-core-2.46.0:2
+			( app-accessibility/at-spi2-atk dev-libs/atk )
+		)
+	app-crypt/libsecret
 	dev-libs/expat
-	dev-libs/glib:2
+	dev-libs/glib
```

Comment:
```suggestion
	dev-libs/glib:2
```

Reply:
Thanks (again) for another great suggestion @juippis.